grunt/webpack-dev-server.js

```diff
@@ -5,15 +5,19 @@ let clone = require('clone');
 // clone the webpack config to separate configuration of webpack and dev server
 let webpackConfig = clone(require('./webpack').options);
 
-//port for webpack-dev-server
+// port for development server
 const port = +('GA'.split('').reduce((p, c)=> p + c.charCodeAt(), ''));
 
-// enable live reload without a script tag
-webpackConfig.entry.vendor.unshift(`webpack-dev-server/client?http://localhost:${port}`);
```

Comment:
Did you try it?  Live reload works for me.

See https://webpack.github.io/docs/webpack-dev-server.html#inline-mode and the following diff.

```diff
 module.exports = {
   options: {
     port,
+    inline: true, // reload on change
     webpack: webpackConfig,
+    publicPath: '/public/',
   },
```